.github/templates/README.tpl

```diff
@@ -6,7 +6,7 @@
 ## Version
 {{ $version }}
 
-Using [terraform-docs](https://github.com/segmentio/terraform-docs) v0.8.2, which is supported and tested on terraform version 0.11+ & 0.12+ but may work for others.
+Using [terraform-docs](https://github.com/terraform-docs/terraform-docs) `v0.8.2`, which is supported and tested on terraform version 0.11+ & 0.12+ but may work for others.
```

Comment:
I'm not familiar with `.github/templates/` folder, what's the purpose of this file?

Reply:
I was playing around with some other github actions for automating the readme/changelog and its just left over from then. 
 There is still a script (scripts/pre-release.sh) that can be used to run this template file to generate the main README.md